src/components/inputs/FlySelect/FlySelect.tsx

```diff
@@ -175,6 +189,63 @@ export default class FlySelect extends React.Component<IProps, IState> {
 		};
 	}
 
+	onContainerKeyDown (event: any) {
+		let open = this.state.open;
+		let focusedIndex = this.state.focusedIndex;
+		switch (event.key) {
+			case 'Enter':
+				open = true;
+				break;
+			case 'ArrowUp':
+				open = true;
+				if (focusedIndex > 0) {
+					focusedIndex--;
+				}
+				break;
+			case 'ArrowDown':
+				open = true;
+				if (focusedIndex < Object.keys(this.state.optionsFormatted).length - 1){
+					focusedIndex++;
+				}
+				break;
+			case 'Tab':
+				open = false;
+				focusedIndex = 0;
+				break;
+		}
```

Comment:
Fantastic, fantastic work on this component! So excited to see all of this available throughout the app 🙌 

I think the only thing I'd like to eventually figure out is the `space` key. For the default browser select element, the spacebar can also be used to open and select options.

Reply:
It's pretty easy to add `space` key functionality. I'll include that with my updates!

Reply:
Really appreciate it! Just gave it a test drive and it felt great with the space key in there!